README.md

```diff
@@ -95,6 +95,39 @@ GoogleMap(
 }
 ```
 
+#### Customizing a marker's info window
+
+You can customize a marker's info window contents by providing a value to the 
+`content` property.
+
+```kotlin
+Marker(
+    //...
+) { marker ->
+    Text(marker.title ?: "Default Marker Title", color = Color.Red)
+}
+```
+
+If you want to customize the entire info window, use the `infoWindow` property
+in the `MapMarker` element instead. Note that if you provide a composable 
+function for `infoWindow`, `content` will be ignored.
+
+```kotlin
+MapMarker(
+    //...
+    infoWindow = { marker ->
+        // Implement the custom info window here
+        Column {
+            Text(marker.title ?: "Default Marker Title", color = Color.Red)
+            Text(marker.snippet ?: "Default Marker Snippet", color = Color.Red)
+        }
+    }
+) { marker ->
+    // Don't do this! This will be ignored
```

Comment:
Could `MapMarker` only take `infoWindow`, and not `content` at all?

That would make this issue impossible, where the `content` gets ignored since `infoWindow` is used.

They could both call through to a private `Marker` or `MapMarker` function that takes both to share implementation, but there's no external way to have both be non-null then.

Reply:
That can work and would prevent the issue altogether. Since that would mean the `MapMarker` would only accept a single `@Composable` parameter, `content` should remain and `infoWindow` should be removed to stick with the guideline in [Compose UI layouts](https://github.com/androidx/androidx/blob/androidx-main/compose/docs/compose-api-guidelines.md#compose-ui-layouts).

I would like to rename `MapMarker` to accommodate this change though. Perhaps `MarkerInfoWindow`?

Reply:
`MarkerInfoWindow` makes sense! It wouldn't be required, but maybe `Marker` takes no `content`, `MarkerInfoWindow` takes a required `content` for the window, and `MarkerInfoWindowContent` takes a required `content` for the info window content? Not sure if that would be clearer than having the default one have an optional `content`.

Reply:
I like it—way more clear on what `content` is.